.github/workflows/format.yml

```diff
@@ -0,0 +1,24 @@
+name: Scalafmt
+
+permissions: {}
+
+on:
+  pull_request:
+    branches: ['**']
+
+jobs:
+  build:
+    name: Code is formatted
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current branch (full)
```

Comment:
WDYM `(full)`?

Reply:
I just copied this from Pekko, that text may be an artifact from an earlier revision which doesn't apply anymore